ChangeLog.md

```diff
@@ -26,6 +26,7 @@ Release notes:
 Major changes:
 
 * `GHCJS` support is being downgraded to 'experimental'. A warning notifying the user of the experimental status of `GHCJS` will be displayed.
+* Upgrade to Cabal 2.4
```

Comment:
Can you move this to the "Unrelease changes" section?  As part of making the final release, I consolidate the changelogs for all the RCs, but this helps to know what's changed between RCs.